include/gsl/gsl_byte

```diff
@@ -37,10 +37,30 @@
 #pragma push_macro("noexcept")
 #define noexcept /*noexcept*/
 #endif           // _MSC_VER <= 1800
+
+#if _HAS_STD_BYTE 
+
```

Comment:
@CaseyCarter yes.  We are now committed to bump at every new official release.  In fact, we've already bumped :-)